changelogs/head.asciidoc

```diff
@@ -7,6 +7,7 @@ https://github.com/elastic/apm-server/compare/8.15\...main[View commits]
 ==== Bug fixes
 
 - Track all bulk request response status codes {pull}13574[13574]
+- Fix APM Agents configuration cache renewal {pull}13958[13958]
```

Comment:
It is fine to remove this line, as this bugfix will likely be backported to 8.15.
```suggestion
```